api/src/test/java/uk/gov/dhsc/htbhf/claimant/ScheduledServiceIntegrationTest.java

```diff
@@ -270,10 +271,13 @@ private void assertChildTurnsOneOnFirstDayEmailPersonalisationMap(PaymentCycle c
 
     private void assertChildTurnsFourEmailPersonalisationMap(PaymentCycle currentCycle, Map personalisationMap) {
         assertCommonPaymentEmailFields(currentCycle, personalisationMap);
+        boolean isPregnant = currentCycle.getExpectedDeliveryDate() != null && currentCycle.getExpectedDeliveryDate().isAfter(LocalDate.now().minusWeeks(12));
         assertThat(personalisationMap.get(CHILDREN_UNDER_1_PAYMENT.getTemplateKeyName())).asString()
                 .contains(formatVoucherAmount(0)); // no children under one
         assertThat(personalisationMap.get(CHILDREN_UNDER_4_PAYMENT.getTemplateKeyName())).asString()
-                .contains(formatVoucherAmount(4)); // only one child will be under four
+                .contains(formatVoucherAmount(isPregnant ? 0 : 4));
```

Comment:
I'm not sure I'm too keen on the Child Under 4 vouchers depending on whether they are pregnant or not, they are two different concepts and should be unrelated.

Reply:
I've updated this so that it checks for children aged >1 in the list of childrensDob instead.